platform/linux-generic/odp_pool.c

```diff
@@ -722,8 +722,9 @@ int buffer_alloc_multi(pool_t *pool, odp_buffer_hdr_t *buf_hdr[], int max_num)
 	if (odp_unlikely(num_deq)) {
 		/* Temporary copy needed since odp_buffer_t is uintptr_t
 		 * and not uint32_t. */
-		uint32_t data[burst];
+		uint32_t *data;
 
+		data = alloca(burst * sizeof(uint32_t));
```

Comment:
There are some [cautionary comments](https://stackoverflow.com/questions/1018853/why-is-the-use-of-alloca-not-considered-good-practice) regarding the use of `alloca()` at Stackoverflow. I'd prefer to find a better workaround to this issue. Sounds like we should be able to simply bound `num_deq` and size the `data` array to a constant maximum value rather than making it dynamic.